microsoft/PowerToys
# [FileExplorerPreview] Move everything from WebBrowser to WebView2

## Summary of the Pull Request

**FIXED**: Why WIP? Same issue as with Monaco, can't test it properly because of all of the async stuff going on while initializing WebView environment. Looking into this...

**What is this about:**
OPEN QUESTIONS:
1) Do we want context menu? **EDIT: Context menu disabled**
<img width="297" alt="image" src="https://user-images.githubusercontent.com/57057282/162190174-73071a56-4ac4-40f1-97ec-27891c6b5a91.png">
<img width="285" alt="image" src="https://user-images.githubusercontent.com/57057282/162190201-136a685a-9e14-403f-8161-f37bf7a249f7.png">

2) WebView allows opening link in the Preview Pane. I guess we don't want that, I just need to find a way to disable it.
@Aaron-Junker Do you know how to do this? **EDIT: Links are now opened in default browser**
![image](https://user-images.githubusercontent.com/57057282/162190438-7cb107b7-a4f8-4bbb-b9ac-fdd665fa49f7.png)


**What is included in the PR:** 

**How does someone test / validate:** 

- Build installer
- Confirm that all p preview and thumbnail handlers works correctly

## Quality Checklist

- [x] **Linked issue:** #15764 #17181 

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
